tests/sts-test.cpp

```diff
@@ -7,7 +7,8 @@ TEST_GROUP (STSTestGroup) {
         sensirion_i2c_init();
 
         ret = sensirion_i2c_mux_set_single_channel(0x72, 0);
-        CHECK_EQUAL_TEXT(0, ret, "sensirion_i2c_mux_set_single_channel(0x72, 0)");
+        CHECK_EQUAL_TEXT(0, ret,
+                         "sensirion_i2c_mux_set_single_channel(0x72, 0)");
```

Comment:
That doesn't really belong to this commit but should be a fixup of the previous one.